docs/src/main/asciidoc/deploying-to-openshift.adoc

```diff
@@ -19,6 +19,7 @@ To complete this guide, you need:
 * JDK 1.8+ installed with `JAVA_HOME` configured appropriately
 * Apache Maven {maven-version}
 * access to an OpenShift cluster (Minishift is a viable option)
+* OpenShift CLI
```

Comment:
We should probably mention that this is optional

Reply:
OpenShift CLI is required for running oc commands from a terminal.  A lot of tasks, not all,  could be achieved using the Web Console instead of using CLI but a developer needs to be comfortable with using CLI in my view.

Reply:
Yeah I know, my point is that the CLI is only needed when you want to do perform the manual deployment

Reply:
Understood. I have updated line #22 to as follows:

* OpenShift CLI (Optional. Only required for manually deploying)

Reply:
Looks great1